src/modules/storage.js

```diff
@@ -0,0 +1,5 @@
+export default class StorageManager {
+  static save = (tasks) => {
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  };
+}
```

Comment:
- [ ] **Required:** There should also be a getter function for getting data from the local storage before feeding it to your `controlDisplay` to `displayTask`. This is the reason why every time the page reloads the previous data although there cannot be rendered to the page.

Reply:
@brytebee Thank you I did what you said please take a re-review.